cmd/dinghy.go

```diff
@@ -109,14 +116,16 @@ func Setup() (*logr.Logger, *web.WebAPI) {
 	var logEventsClient logevents.LogEventsClient
 	var persitenceManager dinghyfile.DependencyManager
 	var persitenceManagerReadOnly dinghyfile.DependencyManager
+
 	// Full SQL mode
-	if config.SQL.Enabled  && !config.SQL.EventLogsOnly {
+	// TODO ensure this is correctly loaded when multi-tenant is used
```

Comment:
Is this still a todo? Is there a test we can write to validate this?

Reply:
Done ✅ !!